binder/applying_corrections.ipynb

```diff
@@ -491,13 +508,73 @@
    "execution_count": 12,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/saransh/Code/HEP/coffea/src/coffea/lookup_tools/dense_mapped_lookup.py:31: NumbaDeprecationWarning: \u001b[1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.\u001b[0m\n",
+      "  out = numba.jit()(feval)\n",
+      "/Users/saransh/Code/HEP/coffea/src/coffea/lookup_tools/dense_mapped_lookup.py:31: NumbaDeprecationWarning: \u001b[1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.\u001b[0m\n",
```

Comment:
I could not find the origin of these warnings. Could you please let me know if they can be silenced or fixed?

Reply:
I've changed it to use `nopython=True` - can you give it a try?

Reply:
That worked, thanks!